javascript/unit-tests/Store.test.ts

```diff
@@ -100,20 +100,6 @@ export function testStore(implName: string, storeMaker: StoreMaker, replacer?: S
         expect((<BundleBuilder>BundleBuilder.deserializeBinary(sent[3])).getTimestamp()).toBe(NEXT_TS2);
     });
 
-    it(`${implName} test claim chains`, async () => {
-        const actorId = 17;
-        await store.claimChain(MEDALLION1, START_MICROS1, actorId);
-        await store.claimChain(MEDALLION2, START_MICROS2, actorId);
-        if (replacer) {
-            await store.close();
-            store = await replacer();
-        }
```

Comment:
claimChain is private now. I am thinking this test is covered by some of the other unit/integration tests we have, but I can figure out a new way to implement it if that would be best.